modules/ztp-preparing-the-hub-cluster-for-ztp.adoc

```diff
@@ -10,9 +10,11 @@ You can configure your hub cluster with a set of ArgoCD applications that genera
 
 .Prerequisites
 
-* Openshift Cluster 4.11 as the hub cluster
-* {rh-rhacm-first} Operator 2.5 installed on the hub cluster
-* Red Hat OpenShift GitOps Operator 1.5 on the hub cluster
+* {product-title} {product-version} or later installed as the hub cluster
+
+* {rh-rhacm-first} 2.3 or later installed on the hub cluster
+
+* {gitops-title} 1.3 or later on the hub cluster
```

Comment:
In the pending docs update there is now a table which includes the recommended software versions. Rather than calling out specific versions here can we simply call out the pre-requisite components and refer to that table for recommended versions?
(the table is in [modules/ztp-telco-ran-software-versions.adoc](https://github.com/openshift/openshift-docs/pull/49805/files#diff-703de89753a4d35a4b3381cf001fc4ad609364909df37e0972d33d14fe8fab26))
something like: 
* {product-title} hub cluster with {rh-rhacm-first} installed. Recommended versions are found in <table reference>

Reply:
Sure. I guess we'll be cherry-picking that work back to 4.10.